pom.xml

```diff
@@ -111,6 +111,7 @@
         <version.joda-time>2.7</version.joda-time>
         <version.jsoup>1.8.1</version.jsoup>
         <version.net.jcip>1.0</version.net.jcip>
+        <version.org.antlr-34>3.4</version.org.antlr-34>
```

Comment:
+1
And eventually Hibernate ORM will use the same version, then we won't need to use a version at all.